internal/manifest/manage.go

```diff
@@ -27,18 +32,98 @@ func Create(path string) error {
 	return filesystem.WriteFile(path, formattedInitManifest)
 }
 
-// Load loads a manifest from a JSON file, unmarshals it, and returns a Manifest object.
-func Load(path string) (*Manifest, error) {
+// Loads, validates, and unmarshals the manifest file at the given path into a flagset
+func LoadFlagSet(manifestPath string) (*flagset.Flagset, error) {
 	fs := filesystem.FileSystem()
-	data, err := afero.ReadFile(fs, path)
+	data, err := afero.ReadFile(fs, manifestPath)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error reading contents from file %q", manifestPath)
 	}
 
-	var m Manifest
-	if err := json.Unmarshal(data, &m); err != nil {
+	validationErrors, err := Validate(data)
+	if err != nil {
 		return nil, err
+	} else if len(validationErrors) > 0 {
+		// TODO tease running manifest validate command
+		return nil, errors.New(FormatValidationError(validationErrors))
+	}
+
+	var flagset flagset.Flagset
+	if err := json.Unmarshal(data, &flagset); err != nil {
+		return nil, fmt.Errorf("error unmarshaling JSON: %v", validationErrors)
+	}
+
+	return &flagset, nil
+}
+
+func Write(path string, flagset flagset.Flagset) error {
+	m := &initManifest{
+		Schema: "https://raw.githubusercontent.com/open-feature/cli/refs/heads/main/schema/v0/flag_manifest.json",
```

Comment:
```suggestion
		Schema: "https://raw.githubusercontent.com/open-feature/cli/main/schema/v0/flag_manifest.json",
```

For Github's raw URLs the `/refs/heads/` is an internal reference. Since it the `Create` and the `openfeature.yaml` have one without the `/refs/heads/` it pulls correct for the command but when it writes it with this one that schema URL ends up with a 404.